doc/administrator/mapfile.rst

```diff
@@ -268,7 +268,7 @@ as follows:
         "default_role_ids" "-1"
         "role_ids" "^-?[0-9]*$$"
 
-The mapfile should be a ``.map.mako`` file, for the variable to be
+The mapfile should be a ``.map.tmpl`` file, for the variable to be
 substituted at make execution time.
```

Comment:
"make execution time" is no longer correct, if I understand correctly? Now it is at container start?